test/EasyRdf/CollectionTest.php

```diff
@@ -40,13 +40,13 @@
 
 class CollectionTest extends TestCase
 {
-    public function setUp()
+    public function setUp(): void
```

Comment:
Since the minimum PHP is 7.1 void return types are supported which makes this change much much easier than it is for projects that still support PHP 7.0. See https://www.php.net/manual/en/migration71.new-features.php